### PR TITLE
Feature: Added support for hiding the toolbar

### LINKED ISSUE #9348: Feature: Add support for hiding the Toolbar
### What's the Problem?

[ FEATURE # 1 ]
I would like an option to hide the tool bar that has the New button. I will never use this because all of those features are available in either the context menu or hotkeys so the option to hide this would make more sense and give more screen space.

### Solution/Idea

[ Feature # 1 ]
Settings toggle to hide the directory toolbar that contains basic context menu entries such as New, Cut, and Paste

### Alternatives

Continue using Windows explorer

### Priorities

| MUST |
All of these features are required for me to efficiently use a file manager.
Screen space is extremely important.

### Files Version

2.3.0.0

### Windows Version

Windows 11 Pro 21H2

### Comments

_No response_

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**

To prevent extra work, all changes to the Files codebase must link to an approved issue marked as `Ready to build`. Please insert the issue number following the hashtag with the issue number that this Pull Request resolves.
- Closes #9348

**Steps used to test these changes**

Stability is a top priority for Files and all changes are required to go through testing before being merged into the repo. Please include a list of steps that you used to test this PR.

1. Confirmed the "Show toolbar" setting is followed
2. Confirmed changing the setting updates the toolbar without the need to restart Files

**Screenshots**

image
image

### CODE REVIEW

File: src/Files.App/Styles/PathIcons.xaml [hunk 1]
```diff
@@ -121,6 +121,8 @@
...
...9255 18.2536 16.4795 18.4521 C 16.0335 18.6507 15.5664 18.75 15.0781 18.75 Z M 15.0488 17.5 C 15.3743 17.5 15.6852 17.4333 15.9814 17.2998 C 16.2777 17.1663 16.5381 16.9873 16.7627 16.7627 C 16.9873 16.5381 17.1663 16.2777 17.2998 15.9814 C 17.4333 15.6852 17.5 15.3743 17.5 15.0488 L 17.5 4.9512 C 17.5 4.6257 17.4333 4.3148 17.2998 4.0186 C 17.1663 3.7223 16.9873 3.4619 16.7627 3.2373 C 16.5381 3.0127 16.2777 2.8337 15.9814 2.7002 C 15.6852 2.5667 15.3743 2.5 15.0488 2.5 L 4.9512 2.5 C 4.6257 2.5 4.3148 2.5667 4.0186 2.7002 C 3.7223 2.8337 3.4619 3.0127 3.2373 3.2373 C 3.0127 3.4619 2.8337 3.7223 2.7002 4.0186 C 2.5667 4.3148 2.5 4.6257 2.5 4.9512 L 2.5 15.0488 C 2.5 15.3743 2.5667 15.6852 2.7002 15.9814 C 2.8337 16.2777 3.0127 16.5381 3.2373 16.7627 C 3.4619 16.9873 3.7223 17.1663 4.0186 17.2998 C 4.3148 17.4333 4.6257 17.5 4.9512 17.5 Z M 5.625 6.25 C 5.4557 6.25 5.3092 6.1882 5.1855 6.0645 C 5.0618 5.9408 5 5.7943 5 5.625 C 5 5.4557 5.0618 5.3092 5.1855 5.1855 C 5.3092 5.0618 5.4557 5 5.625 5 L 14.375 5 C 14.5443 5 14.6908 5.0618 14.8145 5.1855 C 14.9382 5.3092 15 5.4557 15 5.625 C 15 5.7943 14.9382 5.9408 14.8145 6.0645 C 14.6908 6.1882 14.5443 6.25 14.375 6.25 Z Z Z</x:String>
```
Comment (0x5bfa) [inline src/Files.App/Styles/PathIcons.xaml:124]: Is this WebAsset ok?

File: src/Files.App/Strings/en-US/Resources.resw [hunk 2]
```diff
@@ -3859,4 +3859,7 @@
   <data name="StatusWithColon" xml:space="preserve">
     <value>Status:</value>
   </data>
+  <data name="ShowToolbar" xml:space="preserve">
+    <value>Show toolbar</value>
```
Comment (0x5bfa) [inline src/Files.App/Strings/en-US/Resources.resw:3863]: Show inner navigation toolbar
Or let's make a description under the title having this text.

File: src/Files.App/Styles/PathIcons.xaml (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Styles/PathIcons.xaml:124]: It's the closest match that we found

File: src/Files.App/Strings/en-US/Resources.resw (hunk 2 above)
Comment (yaira2) [reply src/Files.App/Strings/en-US/Resources.resw:3863]: "inner navigation" is a coding concept, I don't think users are familiar with it.

**Review by 0x5bfa — APPROVED** [review]

**Pull request merged**